polaris-react/src/components/BulkActions/BulkActions.tsx

```diff
@@ -41,10 +55,10 @@ export interface BulkActionsProps {
   onSelectModeToggle?(selectMode: boolean): void;
   /** Callback when more actions button is toggled */
   onMoreActionPopoverToggle?(isOpen: boolean): void;
-  /** If the BulkActions is currently sticky in view */
-  isSticky?: boolean;
-  /** The width of the BulkActions */
-  width: number;
```

Comment:
Just a note that we will need to update [this bit of code in web](https://github.com/Shopify/web/blob/adeb8a4652a7f5315977d5ef58d9191b929ff3b0/app/shared/components/ResourceListWithHeader/ResourceListWithHeader.tsx#L706-L714)

Reply:
@mrcthms is there more context for why these are being removed in a minor release and not being deprecated first and then removed in the next major release?